### ISSUE TITLE
 Invalid prop: type check failed for prop "completionTimestamp"

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#2607]
### Observed behavior

When viewing an exam report in Coach at this URL:

```
http://localhost:8000/coach/#/eba8f3c694d0d9cb3a9170c81f7e076d/fb51dae6df7545af8455aa3a0c32048d/exams/1d271e1a668d735b254de7a52920f01c/users/d98b14a36191d92a0e85b277a54c3d62/0/0
```

saw this error a few times;

```
Invalid prop: type check failed for prop "completionTimestamp". Expected Date, got String.
```

### Steps to reproduce

* create and activate exam
* take and submit exam without answering any questions
* deactivate exam and view report

### Context

* Kolibri version: 0.7

**Comment by mrpau-eugene** [comment]
@indirectlylit I was trying to reproduce this issue but I'm getting this console error instead of the one you posted

```
[DEBUG: ../kolibri_exercise_perseus_plugin/kolibri_exercise_perseus_plugin/assets/src/views/index.vue] Error during unmounting of item renderer Error: Minified React error #40; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=40 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at r (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:1)
    at Object.unmountComponentAtNode (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:62)
    at t.componentWillUnmount (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:78)
    at p.unmountComponent (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:83)
    at Object.unmountComponent (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:37)
    at p.unmountComponent (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:83)
    at Object.unmountComponent (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:37)
    at l (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:62)
    at r.perform (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:37)
    at Object.batchedUpdates (exercise_perseus_render_module-0.6.1.dev020171205010816-git.js:84)
```

Used the CK-12 Math channel

**Comment by indirectlylit** [comment]
@rtibbles has been doing some recent work on this component, and may even be deprecating it

let's assign to him

**Comment by christianmemije** [comment]
Fixed in #3369
